funnel/views/proposal.py

```diff
@@ -367,6 +369,16 @@ def transfer_to(self):
             )
         return redirect(self.obj.url_for(), 303)
 
+    @route('toggle_featured', methods=['POST'])
+    @requires_login
+    @requires_permission('move-proposal')
+    def toggle_featured(self):
+        featured_form = forms.Form()
+        if featured_form.validate_on_submit():
+            self.obj.featured = not self.obj.featured
+            db.session.commit()
+        return redirect(self.obj.url_for(), 303)
```

Comment:
This view should not be a toggle. It should request a `featured` variable from the front-end and set it.